ledger/accountdb.go

```diff
@@ -2312,6 +2312,86 @@ func performOnlineAccountsTableMigration(ctx context.Context, tx *sql.Tx, log fu
 	return err
 }
 
+// removeOfflineStateProofID clears the StateProofID field for all offline accounts
+func removeOfflineStateProofID(ctx context.Context, tx *sql.Tx, log func(processed, total uint64)) (err error) {
+
+	var updateAcct *sql.Stmt
+	updateAcct, err = tx.PrepareContext(ctx, "UPDATE accountbase SET data = ? WHERE addrid = ?")
+	if err != nil {
+		return err
+	}
+	defer updateAcct.Close()
+
+	var rows *sql.Rows
+	rows, err = tx.QueryContext(ctx, "SELECT addrid, data FROM accountbase")
```

Comment:
I suggest to optimize the query and select only accounts with normalized balance > 0 or, better, use `performOnlineAccountsTableMigration` in order to make all changes in a single pass.